src/Faker/Provider/pt_BR/Person.php

```diff
@@ -99,4 +99,59 @@ public static function suffix()
     {
         return static::randomElement(static::$suffix);
     }
+
+
+    /**
+     * A random CPF number.
+     * @link http://en.wikipedia.org/wiki/Cadastro_de_Pessoas_F%C3%ADsicas#Validation
+     * @param bool $formatted If the number should have dots/dashes or not.
+     * @return string
+     */
+    public function cpf($formatted = true)
+    {
+        $n = $this->generator->numerify('#########');
+        $n .= $this->verifierDigit($n);
+        $n .= $this->verifierDigit($n);
+
+        return $formatted? vsprintf('%d%d%d.%d%d%d.%d%d%d-%d%d', str_split($n)) : $n;
+    }
+
+    /**
+     * A random CNPJ number.
+     * @link http://en.wikipedia.org/wiki/CNPJ
+     * @link http://pt.wikipedia.org/wiki/CNPJ#Algoritmo_de_Valida.C3.A7.C3.A3o
+     * @param bool $formatted If the number should have dots/slashes/dashes or not.
+     * @return string
+     */
+    public function cnpj($formatted = true)
```

Comment:
Here are the company provider.
https://github.com/fzaninotto/Faker/blob/master/src/Faker/Provider/pt_BR/Company.php

A juridic person is a company, how said on Wikipedia, CNPJ is an identification number to companies and legal entities. I think is more logical move this property to company provider.

> Eg.
> I, as a person, have a CPF
> I, as a company, have a CNPJ

I don't know if it make sense for you. Please, tell me what you think about it.